src/components/RadioButton/RadioButtonGroup.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { forwardRef } from 'react';
+import { bem } from '../../utils';
+import styles from './RadioButtonGroup.scss';
+
+export interface Props {
+    /** The name of the group this radio button belongs to */
+    name: string;
+    /** common onChange handler that will be passed to all children */
+    onChange?: () => void;
```

Comment:
Is `onChange` optional? Does it have some arguments?

Reply:
From the original implementation it doesn't contain isRequired
![image](https://user-images.githubusercontent.com/104063210/202469334-dfb24050-3322-49ae-bf7e-4414890023fd.png)